nautobot/extras/tests/test_templatetags.py

```diff
@@ -13,7 +13,7 @@ class JobButtonsTest(TestCase):
     """Test Rendering of Job Buttons."""
 
     @classmethod
-    def setUpTestData(cls):
+    def setUp(cls):
```

Comment:
This does not have to be a `classmethod` anymore.
Should just be def setUp(self).